views/practice/index.html

```diff
@@ -0,0 +1,139 @@
+{{extend 'admin/instructors.html'}}
```

Comment:
Is this file now moved to views/admin/practice/index.html? If so, please remove this from the git repo with `git rm`